apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -174,6 +179,12 @@ class SectionProgress extends Component {
         className={dashboardStyles.dashboardPage}
         data-testid="section-progress-v1"
       >
+        {allowUserToSelectV2View && (
```

Comment:
Nit: could we move this logic to the `SectionProgressSelector`? That class contains all the logic for the banners and determines whether to show v1 or v2. I feel like this popup fits better there.